test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightValueSerializerTests.cs

```diff
@@ -216,6 +216,47 @@ public void WritingCollectionResourceValueWithPropertiesShouldWrite()
             Assert.Equal(@"[{""City"":""MyCity1""},{""City"":""MyCity2""}]", result);
         }
 
+        [Fact]
+        public void WritingResourceValueAndNestedResourceValueWithSimilarPropertiesShouldWrite()
+        {
+            var complexType = new EdmComplexType("NS", "Address");

```

Comment:
`var` vs explicit types in this file?

Reply:
We can use `var` in tests.
The file uses `var` alot.